src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -177,6 +176,21 @@ function MoneyRequestView({
     const [transactionBackup] = useOnyx(`${ONYXKEYS.COLLECTION.TRANSACTION_BACKUP}${getNonEmptyStringOnyxID(linkedTransactionID)}`, {canBeMissing: true});
     const transactionViolations = useTransactionViolations(transaction?.transactionID);
 
+    const originalTransactionIDFromComment = transaction?.comment?.originalTransactionID;
+    const [originalTransaction] = useOnyx(`${ONYXKEYS.COLLECTION.TRANSACTION}${originalTransactionIDFromComment ?? ''}`, {canBeMissing: true});
+    const isExpenseSplit = useMemo(() => {
+        const {originalTransactionID, source, splits} = transaction?.comment ?? {};
+
+        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+        if ((splits && splits.length > 0) || !originalTransactionID || source !== CONST.IOU.TYPE.SPLIT) {
+            return false;
+        }
+
+        // To determine if it’s a split expense, we check for the presence of `comment.splits` on the original transaction.
+        // Split expenses won’t have `comment.splits`, while split bills will have them.
+        return !originalTransaction?.comment?.splits;
```

Comment:
Can we do the following:

1. Create `function isExpenseSplit(transaction, originalTransaction)` function with this logic.
2. Use it here
3. Refactor `getOriginalTransactionWithSplitInfo` to use this logic

That way we don't end up with two places where this code is written, which would lead to them potentially getting out of sync if we change the logic in one place but not the other.

Reply:
Yea, that's what I meant. Thanks!